flank-scripts/build.gradle.kts

```diff
@@ -156,49 +157,47 @@ tasks.register("download") {
     ant.invokeMethod("get", mapOf("src" to sourceUrl, "dest" to destinationFile))
 }
 
-tasks.register("checkIfVersionUpdated") {
-    group = "verification"
```

Comment:
nit: is it still a verification task? If does I think that was nice to have it added to the group, not very important though

Reply:
added